src/Components/Tabs.tsx

```diff
@@ -0,0 +1,48 @@
+import { useEffect, useState } from 'react';
+import { Link, useParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { Tab } from '../types/Tab';
+
+export const Tabs = () => {
+  const tabs = [
+    { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
+    { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
+    { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
+  ];
+
+  const { tabId = 0 } = useParams();
+  const [selectedTab, setSelectedTab] = useState<Tab | null>(null);
```

Comment:
It can be simple variable, no need to keep it in the state
const selectedTab = tabs.find(tab => tab.id === tabId);